docs/langchain.md

```diff
@@ -0,0 +1,96 @@
+
+## Getting Started
+
+First, you'll need to make sure you have `fixie` installed on your machine with `pip install fixieai`. This will install the Fixie CLI, which you'll need for deploying to Fixie. You'll also need a Fixie account and to authorize the Fixie CLI. You can do both by running `fixie auth`.
+
+Next, you'll need to tell Fixie what function to call to invoke your app. As an example, let's imagine something as simple as a [PromptTemplate](https://python.langchain.com/en/latest/getting_started/getting_started.html):
+
+```python
+from langchain.prompts import PromptTemplate
+from langchain.llms import OpenAI
+from langchain.chains import LLMChain
+
+llm = OpenAI(temperature=0.9)
+prompt = PromptTemplate(
+    input_variables=["product"],
+    template="What is a good name for a company that makes {product}?",
+)
+chain = LLMChain(llm=llm, prompt=prompt)
+chain.run("colorful socks")
+```
+
+In the above example, we're calling `chain.run()` directly inside of our code with a hardcoded value of "colorful socks". This isn't that useful, since it means we can only get company names for colorful socks! Instead, we want a user to be able to provide the variable themselves, and we want Fixie to pass that to us.
+
+To accomplish this, we're going to wrap `chain.run()` inside of a func. Here's our new version:
+
+```python
+# main.py
+
+from langchain.prompts import PromptTemplate
+from langchain.llms import OpenAI
+from langchain.chains import LLMChain
+
+llm = OpenAI(temperature=0.9)
+prompt = PromptTemplate(
+    input_variables=["product"],
+    template="What is a good name for a company that makes {product}?",
+)
+chain = LLMChain(llm=llm, prompt=prompt)
+
+def invoke_chain(input):
+  return chain.run(input)
+```
+
+Instead of calling `chain.run()` directly, we've instead wrapped it inside of a function called `invoke_chain` that takes a single `str` as an input variable. We then pass that string directly to our `chain.run()`.
+
+Now we need to tell Fixie about our Chain. To do this, run `fixie init`. Fixie will ask you a few questions about your agent like its handle (its "name" inside of Fixie) and description (this helps others understand how to use your creation).
+
+The **most important part** is the `Entry Point`. This is where you tell Fixie where to find your function. The format is `name_of_your_python_file:name_of_your_function`.
+
+In our case, our file is `main.py`, so we'll put `main:invoke_chain`.
+
+Once you've done this, Fixie will create an `agent.yaml` file in the same directory. You can edit this any time.
+
+You'll also notice that Fixie automatically created a `requirements.txt` file. If you're not familiar with `requirements.txt`, it's a simple way to define the module requirements for your project. Fixie will automatically install these on deploy. In our example, we need the following setup:
```

Comment:
Maybe link to something like https://learnpython.com/blog/python-requirements-file/?